spec/sheet_spec.rb

```diff
@@ -0,0 +1,7 @@
+require 'spec_helper'
```

Comment:
this file is not really doing anything - so you should remove it, or add unit tests to it.